content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,402 @@
+   installed.
+
+   The :guilabel:`WhatsApp` option **only** appears if the *WhatsApp Integration* module is
+   installed.
+
+.. note::
+   :guilabel:`WhatsApp` templates **cannot** be edited on-the-fly. A separate approval from *Meta*
+   is requried.
+
+Then, either select a communication template from the :guilabel:`Template` drop-down menu.
+
+.. note::
+   These existing communication templates can be modified directly from this field, if necessary, by
+   selecting the chosen template, and clicking the :guilabel:`Internal link ➡️ (right arrow)` icon
+   to the right. Doing so reveals a separate page where users can edit the :guilabel:`Content`,
+   :guilabel:`Email Configuration`, and :guilabel:`Settings` of that particular communication
+   template.
+
+.. tip::
+   To create a new communication template from the :guilabel:`Template` field in the
+   :guilabel:`Communication` tab, start typing in the name of the new template, then click
+   :guilabel:`Create and edit...` from the resulting drop-down menu. Doing so reveals a blank
+   :guilabel:`Create Template` pop-up window, wherein users can create a custom communication
+   template.
+
+Next, determine the :guilabel:`Interval`, :guilabel:`Unit`, and :guilabel:`Trigger` from their
+respective drop-down fields, letting Odoo know when the communication should be sent.
+
+The user enters a desired number into the :guilabel:`Interval` field. Then, select a
+:guilabel:`Unit` from the drop-down menu. The options are: :guilabel:`Immediately`,
+:guilabel:`Hours`, :guilabel:`Days`, :guilabel:`Weeks`, and :guilabel:`Months`.
+
+Then, select an option from the :guilabel:`Trigger` drop-down menu. The options are:
+:guilabel:`After each registration`, :guilabel:`Before the event`, and :guilabel:`After the event`.
+
+The figures in the :guilabel:`Sent` column will populate as communications are sent. And, beside the
+number present in the :guilabel:`Sent` column there are different icons that appear, depending on
+the status of that particular communication.
+
+The status of *Running* is represented by a :guilabel:`three gears` icon. The status of *Sent* is
+represented by a :guilabel:`✔️ (checkmark)` icon. And, the status of *Scheduled* is represented by
+an :guilabel:`⌛ (hourglass)` icon.
+
+To delete any communication from the :guilabel:`Communication` tab, simply click the :guilabel:`🗑️
+(trash can)` icon on the corresponding communication line.
+
+There is no limit to how many communications can be added in the :guilabel:`Communication` tab of an
+event form.
+
+.. _create_events/event_form/questions:
+
+Questions tab
+-------------
+
+In the :guilabel:`Questions` tab of an event form, users can create brief questionnaires for
+registrants to interact with, and respond to, after they register for the event.
+
+These questions can be focused on gathering basic information about the attendee, learning about
+their preferences, expectations, and other things of that nature.
```

Comment:
Consider adding info here about reporting and leadgen rules.
We use these questions internally for such purposes.